### ISSUE TITLE
Request, more than an issue

### ISSUE THREAD

**Issue by lyallp** [nabbi/oubliette-overlay#3]
Zoneminder ebuild for Gentoo.
Thanks for providing this, I appreciate this very much.
However, would you please update /etc/apache2/conf/vhosts/10_zoneminder.conf, so that changes in this file are found by the etc-update command?
Background: I installed zoneminder years ago, before the API came into existence, so my Apache config was setup and ran Zoneminder. However, the API did not work when I installed the 9999 release.
The reason was the API has a new set of Rewrite rules for apache in /usr/share/doc/zoneminder-9999/10_zoneminder.conf.bz2)
Given the ebuild did not update the /etc/apache/conf/vhosts.d/10_zoneminder.conf automatically, nor display the readme-gentoo file, I was completely unaware of this change and spent a few days trying to figure out why my API was not working.

Maybe displaying the readme-gentoo would help, as new installations on gentoo will fail, without performing the steps indicated in this readme.

**Comment by nabbi** [comment]
Hello @lyallp 
Thank you for this feedback. 

I modified https://github.com/nabbi/oubliette-overlay/commit/0fe1f8761e9ff7dfae555608f62d3700a8331aa1 the install script to place this under /etc/apache2/vhosts.d/
This dodoc behaviour appears to be a relic of gentoo's original ebuild, unclear if that was an intentional design.

However, your paths differ from my default apache2 installation, so maybe YMMV yet ?
Happy to append a post-install note as I suspect there might be a legacy-apache2 conf file structure not accounted for.

If your current file exists at /etc/apache2/vhosts.d/10_zoneminder.conf, you should now also see the changes I incorporated from the zoneminder devs provided distros/ubuntu2004/conf/apache2/zoneminder.conf file. Haven't tested these updated parameters extensively...

**Comment by lyallp** [comment]
Please find attached my update to the zoneminder-9999 ebuild and to the apache2 config file.
I put these in my local repository, as I wasn't sure  how active this build was.
I am no expert in ebuild creation, so I would like to leave it to you to have a check of my updates.

Threw out the non-9999 build text. Added installation of apache2 config.
[zoneminder-9999.ebuild.txt](https://github.com/nabbi/oubliette-overlay/files/7449873/zoneminder-9999.ebuild.txt)
Updated apache config to remove commets.
[10_zoneminder.conf.txt](https://github.com/nabbi/oubliette-overlay/files/7449875/10_zoneminder.conf.txt)

Note the apache config is incompatible with apache 2.2
Basically, this is what I am using at the moment, it works with zmninja (iPhone App)

**Comment by nabbi** [comment]
Thanks, I should have that all incorporated. 

I'm running www-misc/zoneminder-1.36.10, would you please report back your results with -9999? It's really the same script... let me know if something still doesn't look right.
`layman -S && emerge -va1 zoneminder`

Provided I have a need to run zoneminder, I'll do my best to maintain an ebuild for it. Feedback like this helps us all.
